app/code/Magento/InventoryApi/Test/Acceptance/Data/ConstData.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<entities xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="../../../../../../vendor/magento/magento2-functional-testing-framework/src/Magento/FunctionalTestingFramework/DataGenerator/etc/dataProfileSchema.xsd">
+    <!--TODO Remove when MQE-498 is implemented-->
```

Comment:
This story already implemented in MFTF 2.1.0
Here is commit: https://github.com/magento/magento2-functional-testing-framework/commit/cce66c42b26e5ed03d6d81afcea142fc303b367a
Please remove this temporary constants.

Reply:
Fixed